packages/block-editor/src/store/private-selectors.js

```diff
@@ -286,3 +286,11 @@ export const hasAllowedPatterns = createSelector(
 export function getLastFocus( state ) {
 	return state.lastFocus;
 }
+
+export function getAllBlockBindingsSources( state ) {
+	return state.blockBindingsSources;
+}
+
+export function getBlockBindingsSource( state, sourceName ) {
+	return state?.blockBindingsSources?.[ sourceName ];
```

Comment:
Optional chaining seems unnecessary here

Reply:
Done in [this commit](https://github.com/WordPress/gutenberg/pull/58085/commits/54c313dc3366bd263c8d67b517813c3e368990f8).